cocos/ui/UISlider.cpp

```diff
@@ -723,6 +744,42 @@ void Slider::copySpecialProperties(Widget *widget)
     }
 }
 
+ResouceData Slider::getBackFile()
```

Comment:
@zilongshanren Yes, most modify by studio is editor only, but @minggo think these function can be part of engine. Please refer comments in https://github.com/cocos2d/cocos2d-x/pull/14372